### ISSUE TITLE
Release v1.2.5

### ISSUE THREAD

**Issue by alchymy** [ccsb-scripps/AutoDock-Vina#290]
What is file "vina_1.2.5_mac_x86_64" as it does not appear to run?  Have an older bin file "vina" 1_1_2 from scripps and that runs.

Thanks
Colin

**Comment by rwxayheee** [comment]
Hi @alchymy 
It runs on macOS Monterey and my model is MacBook Pro (Retina, 15", Mid 2015). 
What is your model and OS?

**Comment by alchymy** [comment]
iMac 27" Intel 2019 running Sonoma 14.4  Will check on my macBook as well.

**Comment by alchymy** [comment]
iMac shows it as a "Kind - Script Editor Document" when downloaded.

**Comment by alchymy** [comment]
macBook is same as yours with Monterey 12.7.4  Download just shows as a Kind  'Document'

**Comment by alchymy** [comment]
The original 1.1.2 vina from scripps downloads appeared as an executable file, did have to "chmod +x vina" to get it to run from a script which passes the parameters to it.

**Comment by diogomart** [comment]
@alchymy does this help? https://github.com/forlilab/tutorials/tree/main/installation-mac#vina-executable

**Comment by rwxayheee** [comment]
> macBook is same as yours with Monterey 12.7.4 Download just shows as a Kind 'Document'

It shows as "Document" to me too. It should still work in Terminal. Mac displays it as a "Document" likely because it doesn't have a specified extension. You could change the icon (appearance) and its Type by adding the extension `.dylib` if you want to make it a Unix Executable to Mac. But it won't affect the content/nature/function of the script

**Comment by alchymy** [comment]
the windows .exe version downloaded as is works fine in a bat file.  Will check the above solution.

**Comment by alchymy** [comment]
Yes adding the dylib extension  and  "./vina_1.2.5_mac_x86_64.dylib" in the .command script works.  Many thanks.

**Comment by diogomart** [comment]
@alchymy if you don't add `.dylib` extension and you execute `./vina_1.2.5_mac_x86_64` what does it print?

**Comment by alchymy** [comment]
If I take .dylib off again it still runs now.  Have got it reading a config file.  

My issue now is getting the receptor / ligand file format from mol / pdb to pdbqt format.  The suggested ADFR tools don't work in macOS now, Windows version installs but starts then just exits.  This is on Windows 11 so not sure if it still runs.

**Comment by rwxayheee** [comment]
Hi @alchymy 

If they're command line tools, it's normal to start and just exit. To not exit instantly, run them in a pre-opened terminal instead of starting from Finder. Adding or changing the extension may change how his file is opened by default but again this shouldn't be a necessary thing. I regret making that comment ><!

Check out the [Meeko](https://github.com/forlilab/Meeko) project for ligand and receptor preparation based on Python libraries: 
https://github.com/forlilab/Meeko

If ADFR is preferred, having a Ubuntu VM on Mac will be helpful. See the following notes I left to my fellow students, to use ADT/ADFR on Mac with Apple Silicon (skip the Rosetta parts if yours is Intel)
https://rwxayheee.github.io/ADFR-Suite-and-ADT-in-Ubuntu-VM-on-Apple-Silicon

See this official guide on using ADFR on **Win10** through WSL: 
https://ccsb.scripps.edu/adcp/windows10/

**Comment by alchymy** [comment]
Have now got Vina running with some sample pdbqt format data from the source code "basic_docking/data" on Windows 11 and macOS. There are also sample files in the Scripps ADFR reducing tutorial.  Autodock Vina 1.1.2 is also available as an integrated extension to "SAMSON 2023 R1 (macOS)" which seems to be able to take MOL and PDB files and pass the data in the correct format to Vina to process - this app has a GUI interface.

**Comment by rwxayheee** [comment]
Closing as resolved.
